src/platforms-plugins.json

```diff
@@ -876,7 +876,8 @@
           "image": "intel-edison-arduino.png",
           "enabled": true,
           "information": [
-            "Hardwas is capable of supporting only 4 PWM outputs. As a result, native bindings do not support PWM on pins 10 and 11."
+            "Hardware is capable of supporting only 4 PWM outputs. As a result, native bindings do not support PWM on pins 10 and 11.",
+            "While Johnny-Five does not support serial comms, you can bind to /dev/ttyFMD1 on the Edison Arduino board using <a href\"https://github.com/EmergingTechnologyAdvisors/node-serialport\">the serialport module</a>."
```

Comment:
> While Johnny-Five does not support serial comm

But it does! I have a branch that's about 50% to add the serialConfig, serialRead and serialWrite methods to Galileo/Edison/Joule-IO

- Instead of using `<a href=`, just use markdown syntax